### PR TITLE
config: update with new noise souce inputs

### PR DESCRIPTION
**Pull request by jrs65**

### CODE REVIEW

**Review by andrerenard — APPROVED** [review]

File: config/chime_science_run_gpu.yaml [hunk 1]
```diff
@@ -1548,12 +1549,12 @@ eigcalc:
 
   # Masking config
   bands_filled: [ [0, 30], [251, 262] ]
-  exclude_inputs: [
-    46,   84,  107,  136,  142,  256,  257,  319,  348,  370,  550,
-    551,  579,  638,  688,  742,  768,  784,  807,  855,  944,  960,
-    971, 1005, 1010, 1020, 1023, 1058, 1166, 1225, 1280, 1281, 1285,
-    1314, 1380, 1521, 1523, 1543, 1642, 1687, 1738, 1792, 1794, 1910,
-    1912, 1943, 1945, 1982, 1984, 1987, 2032, 2034 ]
+  exclude_inputs: [  32,   46,   84,  107,  136,  142,  256,  257,  319,  348,  357,
```
Comment (mondana) [inline config/chime_science_run_gpu.yaml:1552]: where do you get 357 and 369 from and why weren't they included before? I don't just know.

Comment (jrs65) [reply config/chime_science_run_gpu.yaml:1552]: They were in the default bad inputs list, but not in the exclude_inputs list for the eigenvalue code, so I added them at the same time.

File: config/chime_science_run_gpu_no_output.yaml [hunk 2]
```diff
@@ -1548,12 +1548,12 @@ eigcalc:
...
   # Masking config
   bands_filled: [ [0, 30], [251, 262] ]
-  exclude_inputs: [
-    46,   84,  107,  136,  142,  256,  257,  319,  348,  370,  550,
-    551,  579,  638,  688,  742,  768,  784,  807,  855,  944,  960,
-    971, 1005, 1010, 1020, 1023, 1058, 1166, 1225, 1280, 1281, 1285,
-    1314, 1380, 1521, 1523, 1543, 1642, 1687, 1738, 1792, 1794, 1910,
-    1912, 1943, 1945, 1982, 1984, 1987, 2032, 2034 ]
+  exclude_inputs: [  32,   46,   84,  107,  136,  142,  256,  257,  319,  348,  357,
+                    369,  370,  453,  550,  551,  579,  638,  688,  739,  742,  768,
```
Comment (mondana) [inline config/chime_science_run_gpu_no_output.yaml:1552]: where is 739 coming from ?

File: config/chime_science_run_recv.yaml [hunk 3]
```diff
@@ -228,7 +228,7 @@ stacking:
   stack_type: chime_in_cyl
   exclude_inputs: [
     # These are non-CHIME feeds we want to exclude (26m, noise source channels, etc.)
-      46,  142,  688,  944,  960, 1058, 1166, 1225, 1314, 1521, 2032, 2034,
+      32,   46,  142,  688,  944,  960, 1058, 1141, 1166, 1225, 1314, 1521, 1642, 2032, 2034,
```
Comment (mondana) [inline config/chime_science_run_recv.yaml:231]: what happenned to 768

**Review by mondana — COMMENTED** [review]
768 was missing on your chime_science_run_recv, but you added bunch of inputs to the other two files for other reasons I assume.

File: config/chime_science_run_gpu_no_output.yaml (hunk 2 above)
Comment (jrs65) [reply config/chime_science_run_gpu_no_output.yaml:1552]: Same as the above ones, it's in the default bad_inputs list.

File: config/chime_science_run_recv.yaml (hunk 3 above)
Comment (jrs65) [reply config/chime_science_run_recv.yaml:231]: It's an end feed so it's already excluded lower down in the list.

**Pull request merged**